cmd/orb-server/startcmd/start.go

```diff
@@ -277,13 +280,18 @@ func startOrbServices(parameters *orbParameters) error {
 
 	// TODO: For now create key at startup, we need different way of handling this key as orb parameter
 	// once we figure out how to expose verification method (webfinger, did:web)
-	keyID, _, err := km.Create(kms.ED25519Type)
+	keyID, _, err := km.Create(kmsKeyType)
```

Comment:
now that we have support for did:web, key should be created before server start.